ocs_ci/ocs/ocp.py

```diff
@@ -259,11 +259,36 @@ def wait_for_resource(
                             return True
                     elif len(sample) == len(in_condition):
                         return True
-            if to_delete and not sample:
-                return True
 
         return False
 
+    def wait_for_delete(self, resource_name='', timeout=60, sleep=3):
+        """
+        Wait for a resource to reach to a desired condition
+
+        Args:
+            resource_name (str): The name of the resource to wait
+                for (e.g.my-pv1)
+            timeout (int): Time in seconds to wait
+            sleep (int): Sampling time in seconds
+
+        Returns:
+            bool: True in case all resources reached desired condition,
+                False otherwise
+
+        """
+        start_time = time.time()
+        while True:
+            try:
+                self.get(resource_name=resource_name)
+            except CommandFailed as ex:
+                logging.error(ex)
```

Comment:
since you expect the command to fail, logging should be info and not error.

Reply:
correct. Made this change.